django/contrib/admin/models.py

```diff
@@ -85,6 +91,46 @@ def is_change(self):
     def is_deletion(self):
         return self.action_flag == DELETION
 
+    def get_changes(self):
```

Comment:
What do you think about naming this something like `get_change_message()`. I feel like `get_changes()` implies a return type of something like list.